### ISSUE TITLE
First Field is Deleted If The Card Text Starts With The Word 'Front'

### ISSUE THREAD

**Issue by DesertBear** [mportuesisf/tinycards-exporter#3]
Firstly, thank you so much for creating this tool, I ran it in Windows Sandbox and it has saved me a lot of time converting the poorly formatted file that Duolingo sent.

I have now successfully converted all the TinyCard flash cards.  However, I thought I would mention that it missed a couple of words during the conversion process.  It appears the reason for this is as follows:  if the actual card text starts with the word 'Front', it causes the first field to be deleted during conversion (presumably because it conflicts with the names 'Front' and 'Back' in the Duolingo card data – which are removed with regex).

It doesn't matter to me personally, as I have already manually added the few words that were missing, however I'm letting you know in case you decide to repurpose the tool for something else.

I have attached the files below.  Feel free to use the completed FlashCards that are also in the zip file yourself if you wish.

**Files:**
[FlashCards.zip](https://github.com/mportuesisf/tinycards-exporter/files/4873010/FlashCards.zip)

**Screenshot showing empty first field errors when importing CSV file into Anki software:**
![Screenshot A](https://user-images.githubusercontent.com/27423064/86514890-27cfbf00-be0d-11ea-8678-f5289779db16.png)

**Screenshot comparing original CSV file to output CSV file from TinyCards Exporter:**
![Screenshot B](https://user-images.githubusercontent.com/27423064/86514894-30c09080-be0d-11ea-871c-d3689d948f9d.png)

**Entries where first field is empty due to the word 'Front':**
```
Front wing
Il parafango anteriore
italian__cars

Front seat
Il sedile anteriore
italian__cars

Front fog light
La fendinebbia
italian__cars

Front crawl
Lo stile libero
italian__swimming

Front wheel
La ruota anteriore
italian__buses

Front door
Il portone
italian__home_exterior

Frontal
Il frontal
italian__muscles
```

**Comment by mportuesisf** [comment]
Yes, the tool has an extremely simple parser based on a state machine, and it looks for the terms 'Front' and 'Back' in the Duolingo card data as delimiters.  However, it can be much smarter about distinguishing delimiters from actual data, and so I'll give a look at fixing it.  Thanks for the bug report.

**Comment by mportuesisf** [comment]
This issue is fixed in version 1.0.1.
